students/JimJenkins/session01/break_me.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+'''
+Author: Jim Jenkins (dvlupr)
+Date: 10/13/2018
+
+Each function, when called, should cause an exception to happen
+Each function should result in one of the four most common exceptions you’ll find.
+For review: NameError, TypeError, SyntaxError, AttributeError
+
+'''
+# functions that trigger each error
+def fn_nameerror():
+    print nameerror
+
+def fn_typeerror():
+    print('typeerror')
+
+def fn_syntaxerror():
+    print('syntaxerror
+
+def fn_attributeError():
+    print('attributeerror')
```

Comment:
Same thing on these - camel_case and produce actual errors.  Take a look and reach out if you have questions.